docs/proposals/volume-provisioning.md

```diff
@@ -101,6 +98,12 @@ We propose that:
 
 ### Controller workflow for provisioning volumes
 
+0. Kubernetes administator can configure name of a default StorageClass. This
+   StorageClass instance is then used when user requests a dynamically
+   provisioned volume, but does not specify a StorageClass. In other words,
+   `claim.Spec.Class == ""`
```

Comment:
Original docs/proposals/volume-provisioning.md specifies that the provisioned PV can be configured by PVC labels. Concrete implementation is not written yet, but I expect something like
`claim.Labels['kubernetes.io/aws-ebs/zone"] = "us-east-1d"`. The provisioner then evaluates both `claim.Class` and `claim.Labels`. It would work the same for hypothetical encryption key ID or anything that the provisioner implements.